### ISSUE TITLE
Thumbnail not pushed to Geoportal Server from ArcCatalog Publish Client

### ISSUE THREAD

**Issue by slead** [Esri/geoportal-server#227]
I'm finding that the thumbnail for a dataset is not being pushed from ArcCatalog to Geoportal Server. To reproduce:

 - in ArcCatalog, choose the ISO 19139 metadata standard under > ArcCatalog Options
 - in the Description tab, enter metadata for a dataset
 - include the Thumbnail
 - using the Publish Client for ArcCatalog, push this dataset's records to Esri Geoportal Server then Approve the record
 - note that the thumbnail is not shown for this record
 - back in ArcCatalog, use the Export Metadata option to create a stand-alone XML metadata file
 - manually load this into Esri Geoportal Server
 - note that the thumbnail is now available for this metadata entry

<img width="569" alt="screen shot 2016-06-28 at 2 07 29 pm" src="https://cloud.githubusercontent.com/assets/133596/16403551/ccb72868-3d39-11e6-8b2f-6738b67f05ef.png">

**Comment by slead** [comment]
on further testing, I now can't get the thumbnail to appear, even via direct XML upload. But as you can see from the above screenshot, it was working at one stage. Thanks for any advice

**Comment by zguo** [comment]
do you still have this issue? I tried one record with browse graphic, seems to work for me, maybe check Identification > Thumbnail sections in the editor to see if info is lost.

**Comment by slead** [comment]
Yep, I still have this issue. I wonder if it's related to the ArcCatalog metadata export function, which also fails to maintain the thumbnail.

I create a new LYR file pointing to a shapefile on disk, and add a thumbnail (which is also stored locally on disk):

![screen shot 2016-07-08 at 10 18 32 am](https://cloud.githubusercontent.com/assets/133596/16673870/6a8eedf6-44f5-11e6-809d-011bbe439d19.png)

Export to standalone XML file:

![screen shot 2016-07-08 at 10 34 42 am](https://cloud.githubusercontent.com/assets/133596/16674055/a8c91bee-44f7-11e6-8a2b-81216a954450.png)


The [resulting XML file](https://github.com/Esri/geoportal-server/files/353296/Local.Government.Areas.xml.txt) also doesn't mention the thumbnail. Could this be related? Does the Publish Tool use the Export XML tool, or is there another unrelated issue?

**Comment by zguo** [comment]
yes the publish client use the same translator behind the scene.
